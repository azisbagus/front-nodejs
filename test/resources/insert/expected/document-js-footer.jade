!!!
html
    head
        title Layout

    body
        include index
        script(src='jquery-1.7.2.js')
