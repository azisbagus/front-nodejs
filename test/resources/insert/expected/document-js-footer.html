<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8"/>
    <title>Karthik Viswanathan</title>
</head>
<body>
    <script src="http://karthikv.net/js/respond.min.js?v=1316246035"></script>
    <script src="jquery-1.7.2.js"></script>
</body>
</html>
